AdminWebsite/AdminWebsite/ClientApp/src/app/booking/endpoints/endpoints.component.html

```diff
@@ -32,13 +32,26 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">Please complete
             <label class="govuk-label govuk-!-width-one-half" [attr.for]="'displayName' + i"> Display name </label>
             <input [id]="'displayName' + i" class="govuk-input govuk-!-width-three-quarters" type="text" formControlName="displayName" />
             <a
-              *ngIf="endpoints.length > 1"
+              *ngIf="endpoints.length > 1 && !videoHearingService.isHearingAboutToStart()"
```

Comment:
Suggest moving this to the component, perhaps use a `get` and give it a meaningful name - makes the template more readable.